app/helpers/application_helper.rb

```diff
@@ -36,4 +36,28 @@ def filter_by_cataloguers_options
   def super_admin_user?(user_id)
     AdminUser.exists?(gravity_user_id: user_id, super_admin: true)
   end
+
+  def creator_label(submission)
+    content_tag(:span, title: creator_title(submission)) { creator_icon(submission) }
+  end
+
+  def creator_icon(submission)
```

Comment:
I'm intrigued by your emoji choices here 😆.

Reply:
Actually I wasn't happy with these but they were the best I could find. Seriously open to alternatives.

Reply:
Maybe:
Admin: 🛡️ 
User: 👤 
Other: ❓

Reply:
Your Unicode-fu is strong. I adopted most of these (🛡️, 👤, ✉️) and updated screenshots to match.